packages/agent/package.json

```diff
@@ -27,14 +27,12 @@
     "@medplum/hl7": "3.1.8",
     "dcmjs-dimse": "0.1.27",
     "iconv-lite": "0.6.3",
-    "node-windows": "1.0.0-beta.8",
```

Comment:
Yeah, this has been hanging around for a while.  It's ok to remove it, but we may need to bring it back.

Context:  The Medplum "Agent" product is kinda like the 3rd or 4th version, descending from custom proprietary projects that we have custom built for customers.  One of the long term goals is to deprecate all of those old proprietary projects, and standardize everything on the agent.

One of the required features of one of those custom proprietary projects is logging to the Windows Event Log, which is why the `node-windows` dep was here.  

We can revisit that in the future when we want to make the final push to converge everything.

Reply:
Yeah that makes sense, this is good context